index.html

```diff
@@ -241,5 +241,35 @@ <h2>Examples</h2>
         ...
       </pre>
     </section>
+    <section class="informative">
+      <h2>Privacy and Security Considerations</h2>
+      <p>
+        The mechanisms introduced by this specification are only available to an application which
+        has called {{MediaDevices/getDisplayMedia()}}, and where the user selected to capture some
+        [=display surface=] <var>S</var>. This means that the capturing application already has
+        access to all of the pixels visible on <var>S</var>.
```

Comment:
Note that in the other PR I was trying to explain the events are only exposed to a CaptureController object living in the same doc where `getDisplayMedia()` was called and only during the duration of the capture, in order to address questions like 2.13. and 2.17.  from the questionnaire.

Reply:
Ack